key.core/src/main/java/de/uka/ilkd/key/speclang/njml/Translator.java

```diff
@@ -1054,6 +1071,16 @@ public Object visitPrimarySuffixCall(JmlParser.PrimarySuffixCallContext ctx) {
         return result;
     }
 
+    private SLExpression processJmlBuiltInFunction(String name,
+            ImmutableList<SLExpression> params) {
+        // at some point in life, we may want to have a customizable map here.
+        return switch (name) {
+        case "\\array2seq" -> termFactory.translateToJDLTerm(name.substring(1), params);
+        case "\\seq_put" -> termFactory.translateToJDLTerm("seqUpd", params);
```

Comment:
That should probably be unified mit `visitSequenceReplace`. I like this method here since it points into a nice configurable direction.

I think something like the following might be nice, ... eventually:
```
functions {
   \jml Seq array2seq(Heap, Object);
   \jml Seq Any head(Seq);
}
```

Reply:
The tutorial does now not longer contain "seq_put" (well, it did not contain it before?)

Reply:
> That should probably be unified mit `visitSequenceReplace`. I like this method here since it points into a nice configurable direction.

I would rather prefer the following when we finally achieve theories: 

```
\theory JML {
  \from Seq \import seqUpd \as seq_upd
}
```

Everything imported or exported from JML theory is available in the JML namespace.